packages/react/src/AnimationProvider/AnimationProvider.module.css

```diff
@@ -0,0 +1,284 @@
+.Animation {
+  transition: var(--brand-animation-transition-default);
+}
+
+/**
+ * Fade in
+ */
+.Animation--fade-in {
+  opacity: 0;
+}
+
+.Animation--fade-in.Animation--active {
+  animation: fade-in var(--brand-animation-duration-default) var(--brand-animation-easing-default) forwards;
+}
+
+@keyframes fade-in {
+  0% {
+    opacity: 0;
+  }
+  100% {
+    opacity: 1;
+  }
+}
+
+/**
+ * Fade out
+ */
+.Animation--fade-out {
+  opacity: 1;
+}
+
+.Animation--fade-out.Animation--active {
+  animation: fade-out var(--brand-animation-duration-default) var(--brand-animation-easing-default) forwards;
+}
+
+@keyframes fade-out {
+  0% {
+    opacity: 1;
+  }
+  100% {
+    opacity: 0;
+  }
+}
+
+/**
+ * Slide in up
+ */
+.Animation--slide-in-up {
+  opacity: 0;
+}
+
+.Animation--slide-in-up.Animation--active {
+  animation: slide-in-up var(--brand-animation-duration-default) var(--brand-animation-easing-default) forwards;
+}
+
+@keyframes slide-in-up {
+  0% {
+    opacity: 0;
+    transform: translateY(15px);
```

Comment:
Should we extract this value to a primitive?

Reply:
👍 Extracted most of the important values into tokens.